src/Controller/Ast/ProcessAstFormController.php

```diff
@@ -7,24 +7,19 @@
 use Illuminate\Http\RedirectResponse;
 use Illuminate\Routing\Controller;
 use Rector\Website\Entity\AstRun;
```

Comment:
it seems it should be:

```suggestion
use Rector\Website\Ast\Entity\AstRun;
```